### ISSUE TITLE
Boolean not supported by default?

### ISSUE THREAD

**Issue by theasp** [alaisi/postgres.async#22]
Hello, I get the following when I query a row with a boolean.  Should booleans work by default?

```
16-05-01 14:47:57 industrial ERROR [five.server.pg-store:35] - Problem executing SQL
java.lang.Thread.run                         Thread.java:  745
io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run           DefaultThreadFactory.java:  137
io.netty.util.concurrent.SingleThreadEventExecutor$2.run      SingleThreadEventExecutor.java:  116
io.netty.channel.nio.NioEventLoop.run                   NioEventLoop.java:  354
io.netty.channel.nio.NioEventLoop.processSelectedKeys                   NioEventLoop.java:  382
io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized                   NioEventLoop.java:  468
io.netty.channel.nio.NioEventLoop.processSelectedKey                   NioEventLoop.java:  511
io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read         AbstractNioByteChannel.java:  130
io.netty.channel.DefaultChannelPipeline.fireChannelRead         DefaultChannelPipeline.java:  787
io.netty.channel.AbstractChannelHandlerContext.fireChannelRead  AbstractChannelHandlerContext.java:  319
io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead  AbstractChannelHandlerContext.java:  333
io.netty.handler.codec.ByteToMessageDecoder.channelRead           ByteToMessageDecoder.java:  161
io.netty.channel.AbstractChannelHandlerContext.fireChannelRead  AbstractChannelHandlerContext.java:  319
io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead  AbstractChannelHandlerContext.java:  333
io.netty.handler.codec.ByteToMessageDecoder.channelRead           ByteToMessageDecoder.java:  161
io.netty.channel.AbstractChannelHandlerContext.fireChannelRead  AbstractChannelHandlerContext.java:  319
io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead  AbstractChannelHandlerContext.java:  333
com.github.pgasync.impl.netty.NettyPgProtocolStream$4.channelRead          NettyPgProtocolStream.java:  178
com.github.pgasync.impl.netty.NettyPgProtocolStream.lambda$newReplyHandler$26          NettyPgProtocolStream.java:  108
com.github.pgasync.impl.PgConnection.lambda$query$7                   PgConnection.java:  100
com.github.pgasync.impl.Functions.applyConsumer                      Functions.java:   29
com.github.pgasync.impl.PgConnectionPool.lambda$null$12               PgConnectionPool.java:   80
postgres.async/execute!/reify/accept                           async.clj:   48
postgres.async.impl/result->map                            impl.clj:   31
clojure.core/vec                            core.clj:  377
...                                          
clojure.core/map/fn                            core.clj: 2646
postgres.async.impl/result->map/fn                            impl.clj:   32
clojure.core/reduce                            core.clj: 6545
clojure.core.protocols/fn/G                       protocols.clj:   13
clojure.core.protocols/fn                       protocols.clj:   75
clojure.core.protocols/iter-reduce                       protocols.clj:   49
clojure.core/partial/fn                            core.clj: 2516
postgres.async.impl/result->map/row->map                            impl.clj:   29
com.github.pgasync.impl.PgRow.get                          PgRow.java:  217
postgres.async.proxy$com.github.pgasync.impl.conversion.DataConverter$ff19274a.toObject                                          
com.github.pgasync.impl.conversion.DataConverter.toObject                  DataConverter.java:  198
postgres.async.proxy$com.github.pgasync.impl.conversion.DataConverter$ff19274a.toConvertable                                          
postgres.async/create-converter/fn                           async.clj:   15
...                                          
java.lang.IllegalArgumentException: No method in multimethod 'from-pg-value' for dispatch value: BOOL
```

**Comment by alaisi** [comment]
Fix available in release 0.7.0

**Comment by theasp** [comment]
Works for me, thank you!
